.travis.yml

```diff
@@ -129,7 +129,7 @@ script:
     - if [[ $TRAVIS_OS_NAME == linux ]]; then
         cmake $TRAVIS_BUILD_DIR -DBUILD_JAVA=1 -DCMAKE_BUILD_TYPE=Coverage -DCMAKE_INSTALL_PREFIX=$GENOMICSDB_INSTALL_DIR -DLIBCSV_DIR=$TRAVIS_BUILD_DIR/dependencies/libcsv -DGENOMICSDB_RELEASE_VERSION=$GENOMICSDB_RELEASE_VERSION -DMAVEN_QUIET=True -DENABLE_LIBCURL=True;
       elif [[ $TRAVIS_OS_NAME == osx ]]; then
-        cmake $TRAVIS_BUILD_DIR -DBUILD_JAVA=1 -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$GENOMICSDB_INSTALL_DIR -DLIBCSV_DIR=$TRAVIS_BUILD_DIR/dependencies/libcsv -DGENOMICSDB_RELEASE_VERSION=$GENOMICSDB_RELEASE_VERSION -DMAVEN_QUIET=True -DENABLE_LIBCURL=False -DPROTOBUF_LIBRARY=$PROTOBUF_HOME/lib/libprotobuf.dylib -DPROTOBUF_INCLUDE_DIRS=$PROTOBUF_HOME/include -DPROTOBUF_PROTOC_EXECUTABLE=$PROTOBUF_HOME/bin/protoc -DPROTOBUF_PROTOC_LIBRARY=$PROTOBUF_HOME/lib/libprotoc.dylib -DPROTOBUF_LIBRARIES=$PROTOBUF_HOME/lib/libprotobuf.dylib -DOPENSSL_PREFIX_DIR=/usr/local/opt/openssl -DCMAKE_C_COMPILER=/usr/local/opt/mpich/bin/mpicc -DCMAKE_CXX_COMPILER=/usr/local/opt/mpich/bin/mpicxx;
+        cmake $TRAVIS_BUILD_DIR -DBUILD_JAVA=1 -DCMAKE_BUILD_TYPE=Debug -DCMAKE_INSTALL_PREFIX=$GENOMICSDB_INSTALL_DIR -DLIBCSV_DIR=$TRAVIS_BUILD_DIR/dependencies/libcsv -DGENOMICSDB_RELEASE_VERSION=$GENOMICSDB_RELEASE_VERSION -DMAVEN_QUIET=True -DENABLE_LIBCURL=False -DPROTOBUF_LIBRARY=$PROTOBUF_HOME/lib/libprotobuf.dylib -DPROTOBUF_INCLUDE_DIRS=$PROTOBUF_HOME/include -DPROTOBUF_PROTOC_EXECUTABLE=$PROTOBUF_HOME/bin/protoc -DPROTOBUF_PROTOC_LIBRARY=$PROTOBUF_HOME/lib/libprotoc.dylib -DPROTOBUF_LIBRARIES=$PROTOBUF_HOME/lib/libprotobuf.dylib -DOPENSSL_PREFIX_DIR=/usr/local/opt/openssl -DCMAKE_C_COMPILER=/usr/local/opt/mpich/bin/mpicc -DCMAKE_CXX_COMPILER=/usr/local/opt/mpich/bin/mpicxx;
```

Comment:
When compiled in Release mode, unordered_set (hash) gets used in a few places and the order of elements isn't deterministic across systems. In Debug/Coverage mode, set gets used which is ordered. This makes the output byte-wise identical to the golden output. The output is functionally correct in Release mode, just not byte-wise exact.

Reply:
Yes, not sure why it was compiled in Release mode in the first place. We should make this Coverage too at some point so we get MacOS coverage reported to `codecov`.

Reply:
Ah - if we get deterministic order in debug then all that bcftools/vcfdiff stuff could probably be taken out of run.py...